warehouse/accounts/interfaces.py

```diff
@@ -66,7 +66,14 @@ def check_password(user_id, password, *, tags=None):
         checking the password.
         """
 
-    def create_user(username, name, password, is_active=False, is_superuser=False):
+    def create_user(
+        username,
+        name,
+        password,
+        is_active=False,
+        is_superuser=False,
+        is_moderator=False,
```

Comment:
I think we don't need this argument here -- we'll only be upgrading already existing users to moderators.

Come to think of it, I doubt we need the `is_superuser` argument either. Might be best to remove that as well.

Reply:
I removed this from the interface.